client/app/bundles/course/assessment/translations.ts

```diff
@@ -122,6 +122,24 @@ const translations = defineMessages({
     id: 'course.assessments.index.submissions',
     defaultMessage: 'Submissions',
   },
+  inviteToKoditsu: {
+    id: 'course.assessments.index.inviteToKoditsu',
+    defaultMessage: 'Invite users to Koditsu Assessment',
+  },
+  invitingUserToKoditsu: {
+    id: 'course.assessments.index.invitingUserToKoditsu',
+    defaultMessage: 'Inviting users to Koditsu Assessment',
+  },
+  invitingUserToKoditsuSuccess: {
+    id: 'course.assessments.index.invitingUserToKoditsuSuccess',
+    defaultMessage: 'Successful in inviting users to Koditsu Assessment',
```

Comment:
"Successfully invited users to Koditsu Assessment"

By the way are we using Koditsu Exam or Koditsu Assessment terminology in CM?

Reply:
I think for now can stick with Koditsu Exam, and I have changed it accordingly..
even though, last time I talked with them, they said there is a vision to expand Koditsu platform to support not only exam but also some other assessment, just like coursemology.